java/README.md

```diff
@@ -25,61 +48,48 @@ At the moment, the Java client must be built from source.
 Software Dependencies:
 
 - JDK 11+
-- git
-- protoc (protobuf compiler)
-- Rust
+
```

Comment:
Actually, you can remove all text in "Build from source" and forward user to DEVELOPER.md
It should contain all that a developer needs.